app/models/piece.rb

```diff
@@ -1,8 +1,21 @@
 class Piece < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Do we need this association? My understanding is that the the piece belongs to the game, then the game in turn belongs to the user.